### PR TITLE
Add ServiceAccounts to kubemark

### PR DESCRIPTION
**Pull request by gmarek**
cc @yujuhong 

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/36810)

### CODE REVIEW

File: test/kubemark/resources/hollow-node_template.json [hunk 1]
```diff
@@ -109,7 +112,7 @@
 					"command": [
 						"/bin/sh",
 						"-c",
-						"./kubemark.sh --morph=proxy $(CONTENT_TYPE) --v=2 1>>/var/logs/kube_proxy_$(MY_POD_NAME).log 2>&1"
+						"./kubemark.sh --morph=proxy $(CONTENT_TYPE) --v=2 1>>/var/logss/kube_proxy_$(MY_POD_NAME).log 2>&1"
```
Comment (dims) [inline test/kubemark/resources/hollow-node_template.json]: is this change a typo? or by design? :)

**Comment by wojtek-t** [comment]
LGTM

@saad-ali - this is purely testing change (touches only kubemark)

File: test/kubemark/resources/hollow-node_template.json (hunk 1 above)
Comment (gmarek) [inline test/kubemark/resources/hollow-node_template.json]: wrong sed, so typo:)

**Comment by wojtek-t** [comment]
@gmarek - please fix bazel.

**Comment by wojtek-t** [comment]
@gmarek - it seems that something is still broken - kubemark is failing.

**Comment by gmarek** [comment]
Looking.

**Comment by gmarek** [comment]
@k8s-bot kubemark test this issue: #IGNORE
Check if it wasn't a flake.

**Pull request merged**